homeassistant/components/esphome/bluetooth/client.py

```diff
@@ -526,6 +526,7 @@ async def read_gatt_char(
                 The characteristic to read from, specified by either integer
                 handle, UUID or directly by the BleakGATTCharacteristic
                 object representing it.
+            **kwargs: Unused
```

Comment:
This file resolves `D417 Missing argument description in the docstring`
All others refer to `UP038 [*] Use 'X | Y' in 'isinstance' call instead of '(X, Y)'`